Gordon360/Services/MembershipService.cs

```diff
@@ -287,5 +289,31 @@ public MembershipView GetMembershipViewById(int membershipId)
 
             return foundMembership;
         }
+
+        /// <summary>
+        /// Filters out memberships that are private with respect to the given viewer
+        /// </summary>
+        /// <param name="memberships">Enumerable of memberships to filter</param>
+        /// <param name="viewerUsername">username of viewer to whom memberships are/are not private</param>
```

Comment:
```suggestion
        /// <param name="viewerUsername">username of viewer</param>
```
I think this language is a little bit confusing. A simpler description still gives the developer what they need to understand this parameter.